src/main/scala/treadle/executable/Memory.scala

```diff
@@ -417,33 +417,25 @@ object Memory {
       }
     }
 
-    /*
+    /**
       * Makes a read chain of pipeline registers.  These must be ordered reg0/in, reg0, reg1/in ... regN/in, regN
```

Comment:
Maybe this comment should be edited to just saw a chain of pipeline registers

Reply:
What exactly would you like the comment to say?

Reply:
"Makes a chain of ..."